### DISCUSSION TITLE
Start up view (Albums for example)

### DISCUSSION THREAD

**Discussion by rvandet** [music-assistant/support#2667] [Feature requests and ideas]
If i start MA, i would like to specify the view it starts with. Like Albums or Radio stations.
Now it starts with the Music providers etc page.
Is there maybe a way to realize this already?

**Comment by OzGav** [comment]
There is a bug currently causing MA to start on the settings page.

**Reply by rvandet** [reply]
I saw that Bug before (and it got solved before).
Any chance that choosing the start page becomes a feature ...

**Reply by OzGav** [reply]
That will depend on many votes this gets (and Marcels priorities)

**Comment by OzGav** [comment]
Now that MA always exposes the webserver you can achieve this by http://192.168.1.166:8095/#/albums

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
